framework/tests/h1h2checker/Issue282.java

```diff
@@ -0,0 +1,17 @@
+import org.checkerframework.framework.testchecker.h1h2checker.quals.*;
+
+public class Issue282 {
+    // Constructor type are not consistent with declared type.
```

Comment:
Can you clarify? Should it be `Declared constructor type is not consistent with default from class`? Or what is it inconsistent with?